EIPS/eip-4972.md

```diff
@@ -0,0 +1,137 @@
+---
+eip: TBD
+title: Name Owned Account
+author: Undisclosed Author <IronchainDAO@gmail.com>, Qi Zhou (@qizhou)
+discussions-to: TBD
+type: Standards Track
+category: ERC
+status: Draft
+created: 2022-04-04
+requires: 20, 721
+---
+
+## Abstract
+This ERC proposes a new type of account - name-owned account (NOA) that is controlled by the owner of the name besides existing externally-owned account (EOA) and contract account (CA). With the new account type, users will be able to transfer/receive tokens using the name-derived address directly instead of the address of the name owner. A NOA can be as a social identity with all states on-chain even under 3rd-party or self custody. It also simplifies porting the social identity from one custody to another.
+
+## Motivation
+A popular way to onboard Web2 users to the Web3 world is custody. However, current custody models have severe drawbacks. Considering the following widely adopted custody models:
+1. The custodian uses one EOA/CA to hold the assets of all users. This is not compatible with on-chain social protocols since all user activities are off-chain.
+2. One EOA per user. The social identity is not portable, which means there is no way for users to migrate their social identity across different custody platforms.
+3. One CA (e.g. Gnosis Safe) per user. The one time deployment cost is super high and the user experience is not good.
+
+To solve all these problems, this ERC proposes a new type of account - name-owned account (NOA). An NOA has
+1. a name for social identity; and
+2. an address derived from the name to receive tokens; and
+3. an owner/owners of the name that is able to transfer the token.
+
+The name should be human-readable and can be easily recognized socially. An example is the username of a centralized platform such as FB, Twitter. The name-derived address (NDA) is a normal Ethereum address that should not collide with the existing addresses of EOA/CA. Since we cannot use NDA as msg.sender, the right to transfer the tokens of the NDA is controlled by the owner/owners of the name. The name to owner/owners mapping is managed by an on-chain name service, and the owner/owners are EOA/CA, which can be the addresses of 3-rd custodians (e.g. FB) or self-custodian. By changing the owner of the name to the EOA of the user (can be done by requesting the custodian), the NDA becomes self-custodian, and no one should be able to transfer the assets unless the approved by the self-custodian user. 
```

Comment:
This paragraph seems more like specification than motivation.

Reply:
Thanks for the comment.  The NOA part is now moved to the "Specification" section.